src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -586,6 +587,93 @@ const formConfig = {
           itemFilter: item => _.get('view:selected', item),
           arrayPath: 'disabilities',
           depends: (formData, index) =>
+            !isProd &&
+            _.get(
+              `disabilities.${index}.view:selectableEvidenceTypes.view:privateMedicalRecords`,
+              formData,
+            ),
+          uiSchema: {
+            disabilities: {
+              items: {
+                'ui:title': disabilityNameTitle,
+                'ui:description': privateRecordsChoice,
+                'view:uploadPrivateRecords': {
+                  'ui:title':
+                    'Do you want to upload your private medical records?',
+                  'ui:widget': 'radio',
+                  'ui:options': {
+                    labels: {
+                      yes: 'Yes',
+                      no: 'No, please get them from my doctor',
+                    },
+                  },
+                },
+                'view:patientAcknowledgement': {
+                  'ui:title': ' ',
+                  'ui:help': patientAcknowledgmentText,
```

Comment:
Oh interesting. I hadn't seen `'ui:help'` before. Cool!

Where did you learn about this? I did a quick [search in USFS](https://github.com/usds/us-forms-system/search?q=ui%3Ahelp&unscoped_q=ui%3Ahelp) and didn't come up with anything helpful.

Reply:
I believe it was Mark Greenburg that showed me that.